### PR TITLE
3.0 follow last changes

### PR DESCRIPTION
**Pull request by cake17**

### CODE REVIEW

File: fr/contributing/documentation.rst [hunk 1]
```diff
@@ -369,15 +370,16 @@ Chaque directive remplit l'index, et l'index des espaces de nom.
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Sphinx will output warnings if a function is referenced in multiple files. It's
-a great way to ensure that you did not add a function two times, but
-sometimes, you actually want to write a function in two or more files, eg.
-`debug object` is referenced in `/development/debugging` and in
-`/core-libraries/global-constants-and-functions`. In this case, you can add
-``:noindex:`` under the function debug, and the warnings will stop to alert you::
+Sphinx va afficher des avertissements si une fonction est référencée dans
+plusieurs fichiers. C'est un bon moyen de s'assurer que vous n'avez pas ajouter
```
Comment (antograssiot) [inline fr/contributing/documentation.rst]: ajouté

File: fr/development/routing.rst [hunk 2]
```diff
@@ -866,6 +866,31 @@ utilisée par ``connect()``::
         ];
     });
 
+Inflection de l'URL pour les Routes Ressource
+---------------------------------------------
+
+Par défaut pour les controllers à plusieurs mots le frament d'URL est la
```
Comment (antograssiot) [inline fr/development/routing.rst]: fragment

File: fr/development/routing.rst [hunk 3]
```diff
@@ -866,6 +866,31 @@ utilisée par ``connect()``::
...
+        ];
+    })
+
+Ce qui est au-dessus va générer des URLs de style ``/blog-posts/*``.
+
+.. note::
+
+    Depuis CakePHP 3.1, le squelette de l'app officiel utilise ``DashedRoute``
+    comme classe de route par défaut. Donc en utilisant l'option
+    ``'inflect' => 'dasherize'`` connecter les routes resssource est recommandé
```
Comment (antograssiot) [inline fr/development/routing.rst]: peut être reformuler la phrase en commençant par "Il es recommandé de..." car la tournure est un peu étrange je trouve

**Comment by cake17** [comment]
c'est corrigé :smile:

File: fr/contributing/documentation.rst [hunk 4]
```diff
@@ -369,15 +370,16 @@ Chaque directive remplit l'index, et l'index des espaces de nom.
...
-`debug object` is referenced in `/development/debugging` and in
-`/core-libraries/global-constants-and-functions`. In this case, you can add
-``:noindex:`` under the function debug, and the warnings will stop to alert you::
+Sphinx va afficher des avertissements si une fonction est référencée dans
+plusieurs fichiers. C'est un bon moyen de s'assurer que vous n'avez pas ajouté
+une fonction deux fois, mais parfois vous voulez en fait écrire une fonction
+dans deux ou plusieurs fichiers, par exemple `debug object` est référencé dans
+`/development/debugging` et dans
+`/core-libraries/global-constants-and-functions`. Dans ce cas, vous pouvez
+ajouter ``:noindex:`` sous la fonction debug pour supprimer les avertissements::
```
Comment (HavokInspiration) [inline fr/contributing/documentation.rst]: General question : the ``:noindex:`` must be added only under one function / method ? Or under both ? I mean, if I have two references to the same function in different part of the doc, I write one method without the ``:noindex:`` and one with it, that's it ?

**Comment by HavokInspiration** [comment]
Other than my question, this looks good.

File: fr/contributing/documentation.rst (hunk 4 above)
Comment (cake17) [inline fr/contributing/documentation.rst]: Yes, your last sentence is correct !
You have to keep only one of the methods **WITHOUT** `:noindex:`. I'll try to make it more accurate

Comment (cake17) [inline fr/contributing/documentation.rst]: Would this be better if I add this sentence at the end of the note:

    suppress warnings.... Keep only one reference **without** ``:no-index:`` to still have the function referenced::

Comment (HavokInspiration) [inline fr/contributing/documentation.rst]: I think that would be better yes :+1:

**Pull request merged**